website/docs/r/function_node.html.markdown

```diff
@@ -11,57 +11,63 @@ description: |-
+- `annotation` - (Optional) Annotation of the Function Node object. Type: String.
+- `description` - (Optional) Description of the Function Node object. Type: String.
+- `func_template_type` - (Optional) Function Template type of the Function Node object. Allowed values: "OTHER", "FW_TRANS", "FW_ROUTED", "CLOUD_VENDOR_LB", "CLOUD_VENDOR_FW", "CLOUD_NATIVE_LB", "CLOUD_NATIVE_FW", "ADC_TWO_ARM", "ADC_ONE_ARM". Default value: "OTHER". Type: String.
+- `func_type` - (Optional) A function type. A GoThrough node is a transparent device, where a packet goes through without being addressed to the device, and the endpoints are not aware of that device. A GoTo device has a specific destination. Allowed values: "GoThrough", "GoTo", "L1", "L2", "None". Default value: "GoTo". Type: String.
+- `is_copy` - (Optional) If the device is a copy device. Allowed values: "yes", "no". Default value: "no". Type: String.
+- `managed` - (Optional) Specified if the function is using a managed device. Allowed values: "yes", "no". Default value: "yes". In ACI version: 5.2 and greater, `managed` is not supported so we need to set it to "no". Type: String.
+- `name_alias` - (Optional) Name alias of the Function Node object. Type: String.
+- `routing_mode` - (Optional) Routing mode of the Function Node object. Allowed values: "Redirect", "unspecified". Default value: "unspecified". Type: String.
+- `sequence_number` - (Optional) Internal property incremented when aaa user logs in. Type: String.
+- `share_encap` - (Optional) Enables encap sharing on node. Allowed values: "yes", "no". Default value: "no". Type: String.
+- `l4_l7_device_interface_consumer_name` - (Optional) The device interface is used to map with a service graph Function Node Connector consumer object. Type: String.
+- `l4_l7_device_interface_provider_name` - (Optional) The device interface is used to map with a service graph Function Node Connector provider object. Type: String.
+- `relation_vns_rs_node_to_abs_func_prof` - (Optional) Represents the relation to L4-L7 Services Function Profile (class vnsAbsFuncProf). Type: String.
+- `relation_vns_rs_node_to_l_dev` - (Optional) Represents the relation to Logical Device Abstraction (class vnsALDevIf). Type: String.
+- `relation_vns_rs_node_to_m_func` - (Optional) Represents the relation to Meta Function (class vnsMFunc). Type: String.
+- `relation_vns_rs_default_scope_to_term` - (Optional) Represents the relation to Terminal Abstract Class (class vnsATerm). Type: String.
+- `relation_vns_rs_node_to_cloud_l_dev` - (Optional) Represents the relation to Cloud L4-L7 Abstract Devices (class cloudALDev). Type: String.
 
 ## Attribute Reference
 
 - `id` - Dn of the function node.
-- `conn_consumer_dn` - Dn of consumer connection in fuction node.
-- `conn_provider_dn` - Dn of provider connection in fuction node.
+- `conn_consumer_dn` - Dn of the Function Node Connector consumer object.
+- `conn_provider_dn` - Dn of the Function Node Connector provider object.
```

Comment:
We probably should remove the "optional" part of the schema definition for those 2 entries.